### ISSUE TITLE
BBA got this wrong

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#842]
4DDD25AD3E3E88C5F0C622C4A41A
![image](https://github.com/user-attachments/assets/1c84ff3c-dc51-4aad-8a09-7f4cd6157283)

BEN got this wrong because the explanation/definitions in BBA was wrong

After 4C and 4D, it is fine that 4H is a Cue, but now 4S is no longer natural but a cuebid
![image](https://github.com/user-attachments/assets/c45e9d0f-8902-4deb-adf3-0aeb965b3451)

The same for 5C - it is a cuebid in both situatiuons - and a common rule is that a splinter, that cuebid the suit again is void

**Comment by EdwardPiwowar** [comment]
I agree with the exception of 4S, which does not exclude playing spades.
Next BBA
![image](https://github.com/user-attachments/assets/32a34fa5-d6dd-4f35-98a3-57274e2bdf03)

**Comment by ThorvaldAagaard** [comment]
The bidding was by Q-Bridge, and I can't find a hand, where it can be right to bid 4S natural, but over 4C 4S would be to play.
